index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/main.css">
+    <title>smash magazine by Jose & Daniel </title>
+</head>
+    <body>
+        <header class="header-main">
+            <div class="header-box">
+                <div><span class="headerLink">Logo</span></div>
+                <div><span class="headerLink">headerLink</span></div>
+                <div><span class="headerLink">headerLink</span></div>
+                <div><span class="headerLink">headerLink</span></div>
+                <div><span class="headerLink">headerLink</span></div>
+                <div><span class="headerLink">HeaderLink</span></div>
+                <div><input class="search" type="search"></div>
+            </div>
+        </header>
+
+        <main>
+            <div class="first-section">
+                <div class="f1"></div>
+                <div class="f1"></div>
+                <div class="f1"></div>
+                <div class="f1"></div>
+            </div>
```

Comment:
Article tags needed for this specific part and for the other article related parts.